index.htm

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Majed-CV"</title>
+</head>
+
+<body>
+  <header>
+    <h1>Majed Shrendan</h1>
+    <h2>Informations</h2>
+    <ul>
+      <li>birthday <time datetime="1995-01-04">04/01/1995</time></li>
+      <li>Telefon: 00963 940217311</li>
+      <li> <a href="mailto:majedshrndan45@gmail.com?subject=feedback">email me</a> </li>
+      <li>Address: Pfeffer weg 6, 13589 Berlin</li>
+    </ul>
+
+  </header>
+  <main>
+    <section>
+      <h2>Work Experience</h2>
+      <p><time datetime=" 2011-01-01">01-01-2011</time> <time datetime="2015-01-01">01-01-2015</time> Five years working as Drill on copper in Syria</p>
+      <p> <time datetime="2015-01">01-2015/06-2015</time> Working in Turkiy</p>
+    </section>
+    <section>
+      <h2>Education</h2>
+      <p>12 Years Studying in the hiegh School</p>
+      <p> <time datetime="2017-04">04-2017</time>German language course B2 level </p>
+
+    </section>
+
+    <section>
+
+
+
+      <h2>Languages</h2>
+      <ul>
+        <li>Arabic</li>
+        <li>English</li>
+        <li>German</li>
+        <li>Turkish</li>
+
+
+      </ul>
+    </section>
+    <section>
+      <h2>IT Skills</h2>
+      <p>linux, git, HTML5, CSS</p>
+    </section>
+    <h2>Hobbies</h2>
+    <ul>
+      <li>Video Games</li>
+      <li>Travelling</li>
+      <li>Football</li>
+      <li>Languages learning</li>
+    </ul>
+  </main>
+
+  <footer>
+    <p>&copy;coded by Said</p>
```

Comment:
It is nice to use it and would be better if it contains other info about Majed ;)